playground/src/share-view/share-view.hbs

```diff
@@ -0,0 +1,8 @@
+<div class="share-view-intro">
+  <h4>Copy & paste unique URL</h4>
+  <p>This unique URL contains all changes made up to that point. As you make further changes to the code, a new URL will be generated.</p>
+</div>
+<textarea id="share-url" class="share-view-url"></textarea>
```

Comment:
Does it make sense to set this to `disabled`?

Reply:
Nope, I think `disabled` would not let you copy it's content to the clipboard ..? I've added `readonly` instead.